.changeset/dry-tools-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/react-i18n': minor
+---
+
+Introduce DateStyle.DateTime.
```

Comment:
This will go in the changelog, can we provide a better description that mentions what DateStyle.DateTime does?

```suggestion
Add `DateStyle.DateTime`. to format dates in the style `Jun 12, 2022 at 10:34 pm`
```

Or something like that.

Reply:
Used your suggestion for this change, thanks!